tests/unit/legacy/api/test_json.py

```diff
@@ -206,6 +191,142 @@ def test_normalizing_redirects(self, db_request):
         assert resp.headers["Location"] == "/project/the-redirect"
 
 
+@pytest.fixture
+def check_json_release(monkeypatch):
```

Comment:
Would this fixture be better sited in `/conftest.py`, next to the other fixtures I've added?

Is this the 'right way' to go about this?

Reply:
Will be refactoring this into an instance-method fixture within the `TestJSONLatestReleases` class. Best to keep it tightly associated with these tests, I think.